README.md

```diff
@@ -148,32 +148,47 @@ The ``smtSolver`` callback function is used to solve SMT queries generated by
 Solidity's SMTChecker.  If you have an SMT solver installed locally, it can
```

Comment:
Should state what solc-js version this started to work and leave a giant *experimental* disclaimer here that the API can change.